apollo-portal/src/main/resources/static/i18n/zh-CN.json

```
@@ -41,8 +41,9 @@
   "Common.Operation": "操作",
   "Common.Delete": "删除",
   "Component.DeleteNamespace.Title": "删除 Namespace",
-  "Component.DeleteNamespace.PublicContent": "删除 Namespace 将导致实例获取不到此 Namespace 的配置，确定要删除吗?",
-  "Component.DeleteNamespace.PrivateContent": "删除私有 Namespace 将导致实例获取不到此 Namespace 的配置，且页面会提示缺失 Namespace（除非使用管理员工具删除 AppNamespace），确定要删除吗?",
+  "Component.DeleteNamespace.PublicContent": "注意，所有环境的公共 Namespace 都会被删除！这将导致实例以及关联的 Namespace 获取不到此 Namespace 的配置,确定要删除吗?",
+  "Component.DeleteNamespace.PrivateContent": "注意，所有环境的私有 Namespace 都会被删除！这将导致实例获取不到此 Namespace 的配置，确定要删除吗?",
+  "Component.DeleteNamespace.LinkedContent": "注意，当前环境关联的 Namespace 会被删除！这将导致实例获取不到此 Namespace 的配置,确定要删除吗?",
```

Use Chinese character `？`.
```suggestion
  "Component.DeleteNamespace.PublicContent": "注意，所有环境的公共 Namespace 都会被删除！这将导致实例以及关联的 Namespace 获取不到此 Namespace 的配置,确定要删除吗？",
  "Component.DeleteNamespace.PrivateContent": "注意，所有环境的私有 Namespace 都会被删除！这将导致实例获取不到此 Namespace 的配置，确定要删除吗？",
  "Component.DeleteNamespace.LinkedContent": "注意，当前环境关联的 Namespace 会被删除！这将导致实例获取不到此 Namespace 的配置,确定要删除吗？",
```